chess/__init__.py

```diff
@@ -1172,7 +1172,7 @@ def unicode(self, *, invert_color: bool = False, borders: bool = False) -> str:
                 if piece:
                     builder.append(piece.unicode_symbol(invert_color=invert_color))
                 else:
-                    builder.append(u"·")
+                    builder.append("·")
```

Comment:
Maybe you can also replace the the `·` with `⭘` as mentioned in [this issue]( https://github.com/niklasf/python-chess/issues/500).